.ci/ci_functions.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash -eE
+# Preparation a workplace & configs to CI
+function configure() {
+    rm -rf "${NFS_WORKSPACE}" ||:
```

Comment:
Maybe change to more popular w/a with `|| true`?

Reply:
Don't see the changes

Reply:
Sorry, done